packages/ui-components/src/NavButton/NavButton.tsx

```diff
@@ -17,9 +17,9 @@ export class NavButton extends React.PureComponent<Props> {
 
     return (
       <NavLink to={to}>
```

Comment:
Can all `<NavLink />` be in light-apps?

Reason: ui-components potentially can be used by other people, which might not use react-router in their personal projects.